src/main/java/com/github/onsdigital/babbage/publishing/PublishingManager.java

```diff
@@ -159,16 +159,27 @@ private void deleteIndex(String key, String uri, String contentType) {
     }
 
     public PublishInfo getNextPublishInfo(String uri) {
-        BoolQueryBuilder builder = QueryBuilders.boolQuery().filter(QueryBuilders.termQuery("uri", uri));
+
+        BoolQueryBuilder builder = QueryBuilders.boolQuery();
+
+        // Only return dates that are in future
+        builder.should(QueryBuilders.rangeQuery("publishDate").gte(System.currentTimeMillis()));
+
+        // if the uri is the homepage then cache until the next publish time.
+        if (uri != null && uri.length() > 1) {
```

Comment:
Could use StringUtils.isNotBlank() for readability https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-

but thats not enough to stop it going in.